tsl/test/sql/include/compression_test.sql

```diff
@@ -27,6 +27,10 @@ FROM original
 FULL OUTER JOIN decompressed ON (original.row_number = decompressed.row_number)
 WHERE (original.*) IS DISTINCT FROM (decompressed.*);
 
+SELECT
+  (SELECT count(*) FROM (:QUERY)) =
+  (SELECT sum(array_length(_timescaledb_functions.compressed_data_to_array(c, cast(NULL as :TYPE)), 1)) FROM compressed) AS "row count matches between original and compressed_data_to_array";
```

Comment:
Would be good to have some tests with the array output to make sure the array with decompressed data return the correct values